### PR TITLE
Add support for AF_PACKET on linux

### PR DESCRIPTION
**Pull request by hjr3**

### CODE REVIEW

File: src/sys/socket/consts.rs [hunk 1]
```diff
@@ -9,6 +9,7 @@ mod os {
     pub const AF_INET: c_int  = 2;
     pub const AF_INET6: c_int = 10;
     pub const AF_NETLINK: c_int = 16;
+    pub const AF_NETLINK: c_int = 17;
```
Comment (posborne) [inline src/sys/socket/consts.rs]: Should be `AF_PACKET`

Comment (hjr3) [inline src/sys/socket/consts.rs]: Oops, I had to copy/paste the patch and I fat fingered. I will fix shortly

Comment (hjr3) [inline src/sys/socket/consts.rs]: Fixed. Thank you.

**Comment by posborne** [comment]
@homu r+

**Comment by posborne** [comment]
Oops, hadn't accept group membership... Trying again.

**Comment by posborne** [comment]
@homu r+

**Comment by hjr3** [comment]
@posborne still did not appear to work.

**Comment by fiveop** [comment]
@homu r+

**Comment by kamalmarhubi** [comment]
@posborne sorry, I forgot to add you to homu reviewers list. I wonder if it's possible to just put the org name instead of duplicating... anyway should be fixed!

**Comment by posborne** [comment]
No problem, thanks!

**Pull request merged**